template_package/carma_package

```diff
@@ -64,9 +84,56 @@ if [[ $PWD != $(realpath ${target_path}) ]]; then
     exit 2
 fi
 
+# Cleanup alternative node files based on user specified type befoe substitutions
+if [ "$plugin_type" = "strategic" ]; then
```

Comment:
You could streamline this block a bit by inverting the conditionals:

```
if [ "$plugin_type"  = "none" ]; then
   delete the plugin stuff that won't be caught by the next branches
if [ ! "$plugin_type" = "strategic" ]; then
   delete the strategic template
if [ ! "$plugin_type" = "tactical" ]; then
   delete tactical template
```

etc. Little cleaner and less duplication that way.

Reply:
Done